cmd/crio/config.go

```diff
@@ -115,6 +115,9 @@ default_mounts = [
 # pids_limit is the number of processes allowed in a container
 pids_limit = {{ .PidsLimit }}
 
+# disable using a shared PID namespace for containers in a pod
+disable_share_pid_namespace = {{ .DisableSharePIDNamespace }}
```

Comment:
Shoulda spotted this yesterday, but I think this variable and especially the customer facing parameter should be 
disable_shared_pid_namespace   (added 'd' to share).  I'd make all of the variables 'shared' instead of "share".

Reply:
I agree change it to shared.